.github/workflows/test.yml

```
@@ -4,36 +4,35 @@ on:
-        pip install pytest pytest-cov
-        
-        pytest || if [ $? -eq 5 ]; then
-          echo "No tests found. Exiting with 0 (instead of 5)."
-          exit 0
-        else
-          exit $?
-        fi
\ No newline at end of file
+      - name: Run tests with pytest
+        working-directory: ${{ env.BACKEND_DIR }}
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2319:warning:5:8: This $? refers to a condition, not a command. Assign to a variable to avoid it being overwritten

(shellcheck)